test/integration/p11-tool.sh.nosetup

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: BSD-2-Clause
+
+set -x
+
+if [ "${DOCKER_TAG:-nodocker}" = "fedora-30" -a "${CC:-nocc}" = "clang" ]; then
+  echo "p11-tool on fedora with clang is dysfunctional"
+  exit 77
+fi
+
+source "$T/test/integration/scripts/helpers.sh"
+
+tempdir=$(mktemp -d)
+
+function cleanup() {
+	rm -rf "$tempdir"
+}
+trap cleanup EXIT
+
+onerror() {
+  echo "$BASH_COMMAND on line ${BASH_LINENO[0]} failed: $?"
+  exit 1
+}
+trap onerror ERR
+
+setup_asan
+
+if [ -z "$modpath" ]; then
+  modpath="$PWD/src/.libs/libtpm2_pkcs11.so"
+fi
+
+function setup_spy() {
+    export PKCS11SPY="$modpath"
+    export PKCS11SPY_OUTPUT=${0}_pkcs11spy.log
+    modpath=${PKCS11SPY_LIB:-/usr/lib/x86_64-linux-gnu/pkcs11-spy.so}
+}
+
+if [ "${USE_PKCS11SPY:-no}" != "no" ]; then
+    setup_spy
+fi
+
+echo "modpath=$modpath"
+
+function p11_tool() {
+    p11tool -d 9999 $@
+}
+
+export HOME=${tempdir}
+mkdir -p $HOME/.config/pkcs11/modules
+echo "module: $modpath" >$HOME/.config/pkcs11/modules/tpm2-pkcs11.module
+
+export TPM2_PKCS11_STORE="$tempdir"
+
+echo "TPM2_PKCS11_STORE=$TPM2_PKCS11_STORE"
+
+echo "testdata">${tempdir}/data
+
+echo "Test p11-kit for modules"
+p11-kit list-modules || true
+echo "Done"
+
+echo "Should have uninitialized token"
+p11_tool --list-tokens
+if ! p11_tool --list-tokens | grep -q "Model: SW   TPM"; then
+  echo "p11tool did not find this token."
+  echo "If you are running a recent version of p11tool then you need to configure the module system-wide:"
+  echo "echo \"module: $TRAVIS_BUILD_DIR/build/src/.libs/libtpm2_pkcs11.so\" >/etc/pkcs11/modules/tpm2_pkcs11.module"
+  exit 77 #TEST SKIP
```

Comment:
This condition should be a skip?

Reply:
Could be an ERROR (99) if you like ?

Reply:
On fedora this remains a skip, on others it's now an error